src/main/java/org/jboss/xavier/analytics/pojo/output/workload/summary/ScanRunModel.java

```diff
@@ -100,12 +101,25 @@ public void setDate(Date date) {
         this.date = date;
     }
 
-    public String getType() {
-        String typeString = "Virt Platform";
-        return typeString + (type?" + SmartState": "");
+    public Boolean getType() {
+        return type;
```

Comment:
As we agreed on, this change will require to create a Ticket for changing the UI so there are no visual changes. For now, I can confirm that this block of code will change the `Scans run` table of the `Workload Migration summary` report... I'm writing just not to forget to create the ticket

![Screenshot from 2019-12-19 10-35-17](https://user-images.githubusercontent.com/2582866/71168033-2ca89900-2256-11ea-8618-df7b2915950c.png)

Reply:
@jonathanvila and @carlosthe19916 sorry, just for my benefit.
The old `getType` method's implementation has changed: it returns a `Boolean` and not the `String` the UI was showing (highlighted in the above screenshot).
Is there, in this PR, a new method that return the same `String` the old `getType` method provided?

Reply:
No , there is no method to return the string needed. The UI ticket is only going to show the text that comes from the report or it will convert the boolean to the String it was being sent ? When we said the UI was going to change because of this change I assumed it was because the UI would be in charge of that change. If it's the backend who has to send the string in the report , then that method is missing at the moment.

Reply:
Discussed a little bit more about this with @carlosthe19916 because to me having a `boolean type` field was not clear what it referred to.
The previous `String type` was providing the `Type` label value to the shown in the UI (screenshot above) but now a boolean value is hard to understand what it refers to just calling it `type`.
I think it could be improved having the field to be renamed to `smartStateEnabled` so that it's clear the boolean refers to Smart State begin enabled in the analysis.

Then we will change the UI to manage this boolean field accordingly

Reply:
@mrizzi @jonathanvila I've created a PR (on Jonathan's branch) for renaming `type` to `smartStateEnabled`; however, this change is just using `@JsonProperty("smartStateEnabled")` so it won't affect the current database model. It is not trivial to change the database model since hibernate won't update the database automatically.  for us. Maybe in the future, we can introduce a database version control manager like [flyway](https://flywaydb.org/)

Reply:
Before it was also a boolean on the mutator setType , and it was not clear at all too, then we had a String accessor getType not returning the internal value but transforming it using a hard coded string. So we lived with unclearity also before. 

My suggestion is to solve the type of the attribute now and in a separate ticket rename the attribute, and execute the rename into the database or implement Flyway (or any other DB change tool ) in order to move forward with this PR